plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentAccuracyRadiusAttributeParser.kt

```diff
@@ -0,0 +1,29 @@
+package com.mapbox.maps.plugin.locationcomponent
```

Comment:
could we still generate this class?

Reply:
It will need to add new template files and change the generating script in stylegen. Since it is the only parser file like this style, I feel it's not worth generating it.